lib/codit/society/lateral.py

```diff
@@ -15,9 +15,9 @@ def coopt_existing_test(track, notes, person):
 
 class LateralFlowUK(UKSociety):
 
-    DAYS_BETWEEN_REPEATED_TESTS = 1
-    VALENCY_TEST_FREQUENCY_DAYS = 3
-    GENERAL_VALENCY_QUANTILE_THRESHOLD = 0.95
+    DAYS_BETWEEN_REPEATED_TESTS = 4
```

Comment:
adjusting these parameters to be nearer to today's reality in the UK

Reply:
(not important for this review but) better in the long term for these to be params with these as default values rather than hard coded in the file.